### PR TITLE
Change JSON and CSV read double config

### PR DESCRIPTION
**Pull request by GaryShen2008**
set spark.rapids.sql.json.read.double.enabled
and spakr.rapids.sql.csv.read.double.enabled true by default

Close https://github.com/NVIDIA/spark-rapids/issues/5813

Signed-off-by: gashen <gashen@nvidia.com>

### CODE REVIEW

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala [hunk 1]
```diff
@@ -913,7 +913,7 @@ object RapidsConf {
   val ENABLE_READ_CSV_DOUBLES = conf("spark.rapids.sql.csv.read.double.enabled")
     .doc("CSV reading is not 100% compatible when reading doubles.")
     .booleanConf
-    .createWithDefault(false)
+    .createWithDefault(true)
```
Comment (tgravescs) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:916]: do we need to update any tests?  ie tests assumed false so now we only test true

Comment (GaryShen2008) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:916]: The only place to set it explicitly is https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/integration_tests/src/main/python/csv_test.py#L171.
Every reading test related to double set spark.rapids.sql.csv.read.double.enabled to true explicitly.
I don't find any test case which requires it be false.

Comment (GaryShen2008) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:916]: Same as json_test.py.

Comment (GaryShen2008) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:916]: @tgravescs  Can you review it again?

**Comment by sameerz** [comment]
Is this a duplicate of https://github.com/NVIDIA/spark-rapids/pull/5935 ? 

cc: @razajafri

**Comment by razajafri** [comment]
> Is this a duplicate of #5935 ?
> 
> cc: @razajafri

Yes, looks like it is. You can decide which one to close

**Comment by GaryShen2008** [comment]
Closed because duplicated

**Pull request closed without merging**